_source/logzio_collections/_log-sources/azure-graph.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Ship Microsoft Graph API data to Logz.io
+logo:
+  logofile: graph-api-logo.png
+  orientation: vertical
+data-source: Microsoft Graph API
+data-for-product-source: Logs
+open-source:
+  - title: Microsoft Graph API fetcher
+    github-repo: logzio-api-fetcher
+contributors:
+  - nshishkin
+shipping-tags:
+  - azure
+order: 470
+---
+
+
+Microsoft Graph is a RESTful web API that enables you to access Microsoft Cloud service resources. This integration allows you to collect data from Microsoft Graph API and send it to your Logz.io account.
+
+
+<div class="tasklist">
+
+
+##### Pull the Docker image of the Logz.io API fetcher
+
+```shell
+docker pull logzio/logzio-api-fetcher
+```
+
+
+##### Create a local directory for this integration
+
+You will need a dedicated directory to use it as mounted directory for the Docker container of the Logz.io API fetcher.
+
+```shell
+mkdir logzio-api-fetcher
+cd logzio-api-fetcher
+```
+
+##### Create a configuration file
+
+In the directory created in the previous step, create a file `config.yaml` using the example configuration below:
+
+```yaml
+logzio:
+  url: https://<<LISTENER-HOST>>:8071
+  token: <<LOG-SHIPPING-TOKEN>>
+
+oauth_apis:
+  - type: azure_graph
+    name: azure_test
+    credentials:
+      id: <<AZURE_AD_SECRET_ID>>
+      key: <<AZURE_AD_SECRET_VALUE>>
+    token_http_request:
+      url: https://login.microsoftonline.com/<<AZURE_AD_TENANT_ID>>/oauth2/v2.0/token
+      body: client_id=<<AZURE_AD_CLIENT_ID>>
+        &scope=https://graph.microsoft.com/.default
+        &client_secret=<<AZURE_AD_SECRET_VALUE>>
+        &grant_type=client_credentials
+      headers:
+      method: POST
+    data_http_request:
+      url: https://graph.microsoft.com/v1.0/auditLogs/signIns
+      headers:
+    json_paths:
+      data_date: createdDateTime
+      next_url:
+      data:
+    settings:
+      time_interval: 1
+      days_back_fetch: 30
+  - type: general
```

Comment:
Please remove the general section (from this line to the end of the configuration)